docs/docs-content/clusters/data-center/nutanix/add-nutanix-cloud-account.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_label: "Add Nutanix Cloud Account"
+title: "Add Nutanix Cloud Account"
+description: "Learn how to add a Nutanix cloud account in Palette."
+hide_table_of_contents: false
+tags: ["data center", "nutanix"]
+---
+
+ Once the Private Cloud Gateway (PCG) is installed, the Nutanix cloud account must be added to Palette. The required fields to add the cloud are pre-defined by the System Administrator during the cloud registration process. These cloud-specific fields vary based on the cloud type.
```

Comment:
All other instances of system administrators are lower cased.

Reply:
lower cased this for consistency.